src/Commands/Remote/SSHBaseCommand.php

```diff
@@ -80,14 +81,28 @@ protected function executeCommand(array $command_args)
         $command_summary = $this->getCommandSummary($command_args);
         $command_line = $this->getCommandLine($command_args);
 
-        $ssh_data = $this->sendCommandViaSsh($command_line);
+        // Retrieve the trace ID from the TraceId class
+        $trace_id = TraceId::getTraceId();
 
-        $this->log()->notice('Command: {site}.{env} -- {command} [Exit: {exit}]', [
-            'site'    => $this->site->getName(),
-            'env'     => $this->environment->id,
-            'command' => $command_summary,
-            'exit'    => $ssh_data['exit_code'],
-        ]);
+        // Log the trace ID for user visibility only in debug mode
+        if ($this->output()->isDebug()) {
+            $this->log()->notice(
```

Comment:
What if we log this as `$this->log()->debug()` instead? I think you can even get rid of the if here.

(I may be wrong, please test)

Reply:
Sure, it just won't be a notice.